templates/zerver/development/design_testing/banners.html

```diff
@@ -0,0 +1,211 @@
+{% extends "zerver/base.html" %}
+{% set entrypoint = "dev-testing" %}
+
+{% block title %}
+<title>{{ doc_root_title }} | Zulip Dev</title>
+{% endblock %}
+
+{% block content %}
+<div class="portico-container" data-platform="{{ platform }}">
+    <div class="portico-wrap">
+        {% include 'zerver/portico-header.html' %}
+        <div class="app portico-page">
+            <div class="banner-wrapper" id="dev_navbar_alerts_wrapper"></div>
+            <div class="app-main portico-page-container">
+                <div class="design-testing-wrapper">
+                    <div class="banner-wrapper" id="dev_normal_banner_wrapper"></div>
+                    <section class="design-controls-section">
+                        <div class="design-testing-controls-label">Theme Settings</div>
+                        <div class="design-testing-controls">
+                            <div class="design-testing-control">
```

Comment:
Probably for code reuse sanity reasons, you want to create a `development/design_testing/base.html` base Jinja2 template to share with the other design testing page, that just lets you define sections like `block design_controls` for the `design-control-section`, so you don't find yourself copy-pasting so much HTML and needing to work in such deep nesting.

Reply:
I guess the path we'd want to migrate towards would be something like `development/ui_components_sandbox/` if we go with naming suggested here: https://chat.zulip.org/#narrow/channel/6-frontend/topic/design.20components.20sandbox.20naming/near/2029702.